app/models/concerns/fog_extensions/aws/server.rb

```diff
@@ -36,6 +36,10 @@ def vm_description
       def ip_addresses
         [public_ip_address, private_ip_address].flatten.select(&:present?)
       end
+
+      # HACK: For IAM instance profile override on host creation form
+      def iam_instance_profile_name
```

Comment:
this means we are not able to retrieve currently set profile?
If not, we really need to disable the field. on Host edit

Reply:
Hello,

Yes indeed, I have no clue how to retrieve profile being set, and it seems it was not done anyway using previously method (profile attached to image).